spec/helpers/codelist_helper_spec.rb

```diff
@@ -4,14 +4,14 @@
   describe "version 2_03" do
     let(:version) { "2_03" }
 
-    describe "#yaml_to_objects" do
-      it "raises an error when the YAML file is missing or incorrect" do
-        expect { helper.yaml_to_objects(entity: "generic", type: "favourite_colours") }
+    describe "#codelist_to_objects" do
+      it "raises an error when the codelist is missing or incorrect" do
+        expect { helper.codelist_to_objects(type: "favourite_colours") }
           .to raise_error "CodelistHelper::UnreadableCodelist"
       end
 
-      it "formats the data in a yaml file to an array of objects for use in govuk form builder" do
-        expect(helper.yaml_to_objects(entity: "generic", type: "default_currency"))
+      it "formats the data from a codelinst to an array of objects for use in govuk form builder" do
```

Comment:
Small typo in the description, if you want to fix it: "codelinst" -> "codelist".